src/lib/rule-finder.js

```diff
@@ -108,6 +108,6 @@ function RuleFinder(specifiedFile, options) {
   this.getUnusedRules = () => getSortedRules(unusedRules);
 }
 
-module.exports = function (specifiedFile, options = {}) {
+module.exports = function (specifiedFile, options) {
```

Comment:
I would have thought exactly the other way around, the reason being the exported function only "dispatching" its arguments through, to the original `RuleFinder`. And exactly that is the place where errors would occur, when no `option` parameter would be specified.